juniper/src/schema/model.rs

```diff
@@ -432,17 +480,20 @@ impl<'a, S> SchemaType<'a, S> {
     }
 
     /// Make a type.
-    pub fn make_type(&self, t: &Type) -> TypeType<S> {
-        match *t {
-            Type::NonNullNamed(ref n) => TypeType::NonNull(Box::new(
-                self.type_by_name(n).expect("Type not found in schema"),
+    pub fn make_type(&self, t: DynType) -> TypeType<S> {
```

Comment:
This method is public, but `DynType`/`AsDynType` are not exported, so in the generated rustdocs this method is visible, but I'm unsure whether it's actually usable from outside juniper since you would have a hard time instantiating a `DynType` value?

Should functions that have private(ish) types in their signature be `pub(crate)` instead?